common/djangoapps/track/service.py

```diff
@@ -0,0 +1,35 @@
+"""
+Implementation of "Analytics" service
```

Comment:
Coming into this code without the external context I already have, I'd be a bit confused about what this file is for and how it fits into the overall analytics tracking systems. Perhaps add a couple of sentences explaining?